exercises/storeWithArrays.js

```diff
@@ -71,6 +68,7 @@ for (let user of allUsers) {
   if (userCanAffordSofa(user, sofa)) {
     console.log('Congratulations!');
     console.log(`A ${sofa[0]} costs ${sofa[1]}, which is within your budget of ${user[2]}!`);
+    console.log('We will email you at ' + user[3] + ' shortly!')
```

Comment:
Try to "when in Rome" code you're working on. The other strings use template literals, so you should do the same.